### PR TITLE
Add Alm project structure and transaction form

### PR DESCRIPTION
**Pull request by patitonar**
This PR adds the functionality to search for a tx hash. For now, it only displays the status of the transaction. 

It validates the status of the transaction introduced with the following states:
- Correct with one bridge message
- Correct with a lot of bridge messages (in this case it will show a list of messages to select one)
- Correct but without bridge messages
- Failed transaction
- Does not exists/Pending transaction (in this case it will retry to load the tx receipt in case it is confirmed later) 

The project supports different theme styles. For now, the styles are very simple and we can improve it later after having the core functionality working.

There are 3 URL paths:
- `/` the base path to introduce to search for the tx hash
- `/chainId/txHash` by entering this link it will automatically start the search for that tx hash in the specified chain. In the case of multiple messages, it will display the option to select one.
- `/chainId/txHash/messageIndex` same as the other path but it will also automatically select the specified message of the transaction 

There is a deployed Demo here for Sokol-Kovan AMB bridge: https://alm-demo-e0998.web.app/

Some examples :
- Not found tx https://alm-demo-e0998.web.app/77/0x115921f63c63f176d20ebad1e4650fcd0fbeb9f2923d9c5e8671d534544e35e9
- Failed tx https://alm-demo-e0998.web.app/42/0x115921f63c63f176d20ebad1e4650fcd0fbeb9f2923d9c5e8671d534544e35e9
- Correct but with no bridge messages https://alm-demo-e0998.web.app/42/0x5cae65019be1f88614b86988a998c36400609e1bd6901ddb60b6d88bb1a37af2
- Correct with one bridge message https://alm-demo-e0998.web.app/42/0x8fcb5a268a6e02c554c9c01a7a74243ce5102a4f5053b40b4cd672a0ee612297
- Correct with multiple bridge messages https://alm-demo-e0998.web.app/42/0x642b340a0bb24d378aaf2762c81e48d6f497a6600bc1b8ae7573e8ca019bc657
- Correct with multiple bridge messages and already selected one https://alm-demo-e0998.web.app/42/0x642b340a0bb24d378aaf2762c81e48d6f497a6600bc1b8ae7573e8ca019bc657/2

### CODE REVIEW

File: alm/src/config/constants.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+export const HOME_BRIDGE_ADDRESS: string = process.env.REACT_APP_COMMON_HOME_BRIDGE_ADDRESS || ''
+export const FOREIGN_BRIDGE_ADDRESS: string = process.env.REACT_APP_COMMON_FOREIGN_BRIDGE_ADDRESS || ''
+
+export const HOME_RPC_URL: string = process.env.REACT_APP_COMMON_HOME_RPC_URL || ''
```
Comment (akolotov) [inline alm/src/config/constants.ts:4]: most probably you will need configuration parameters to build links to Etherscan and Blockscout. We have similar ones in the UI's `.env` file.

File: alm/src/config/descriptions.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// %t will be replaced by the time -> x minutes/hours/days ago
+export const TRANSACTION_STATUS_DESCRIPTION: { [key: string]: string } = {
```
Comment (akolotov) [inline alm/src/config/descriptions.ts:2]: For all messages containing the transaction hash (except one that is not found) it makes sense to display the tx id as a link to Blockscout/Etherscan.

File: alm/src/config/constants.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+export const FOREIGN_RPC_URL: string = process.env.REACT_APP_COMMON_FOREIGN_RPC_URL || ''
+
+export const networks: { [key: number]: string } = {
+  1: 'ETH Mainnet',
+  42: 'Kovan Testnet',
+  61: 'Ethereum Classic',
+  77: 'Sokol Testnet',
+  99: 'POA Network',
+  100: 'xDai Chain'
+}
```
[the remark this answers is not in the document]
Comment (patitonar) [reply alm/src/config/constants.ts]: Added the network name as parameters https://github.com/poanetwork/tokenbridge/pull/353/commits/4265cfad993e40c1cabf69bfd17b802ed0fe57a2

File: alm/src/config/constants.ts (hunk 1 above)
Comment (patitonar) [reply alm/src/config/constants.ts:4]: Added variables for tx links template https://github.com/poanetwork/tokenbridge/pull/353/commits/475cf8a338d1d03b2266b100e57fa6a930c78e80

File: alm/src/config/descriptions.ts (hunk 2 above)
Comment (patitonar) [reply alm/src/config/descriptions.ts:2]: Added the links to the explorer https://github.com/poanetwork/tokenbridge/pull/353/commits/dbc7032d65bbb85c1c57889f73fa465d290e1ed7

**Comment by patitonar** [comment]
Addressed the comments, also the demo was updated https://alm-demo-e0998.web.app/

**Pull request merged**